src/HTSPDemuxer.cpp

```diff
@@ -226,6 +226,47 @@ PVR_ERROR CHTSPDemuxer::CurrentSignal ( PVR_SIGNAL_STATUS &sig )
   return PVR_ERROR_NO_ERROR;
 }
 
+int64_t CHTSPDemuxer::GetTimeshiftTime() const
```

Comment:
True, but it shouldn't matter since I doubt the mutex will be locked for more than a split second. We could move the whole algorithm to a method inside CHTSPDemuxer though, then we could lock just once without having to make the mutex public (something I really want to avoid).